src/commands/apply.js

```diff
@@ -0,0 +1,82 @@
+const log = require('@dhis2/cli-helpers-engine').reporter
+const path = require('path')
+
+const { exec } = require('./apply/exec')
+const {
+    findAvailableCodemodsInNodeModules,
+} = require('../utils/codemods/findAvailableCodemodsInNodeModules')
+const {
+    getCodemodByPackageAndName,
+} = require('../utils/codemods/getCodemodByPackageAndName')
+const { makePaths } = require('../utils/makePaths.js')
+
+module.exports.command = 'apply <codemod> <files..>'
```

Comment:
The default option doesn't work for me:

```
varl@veronika:~/dev/dhis2/cli/cli-utils-codemods$ ./bin/d2-utils-codemods apply /home/varl/dev/dhis2/libs/d2/codemods/31.1.0-change-imports.js
d2-utils-codemods apply <codemod> <files..>

Applys a codemod on a given file/directory

Positionals:
  codemod  Colon-separated node package & codemod name. OR a direct path to
           codemod file                                      [string] [required]
  files    Files or path the codemod should be applied to
                                            [array] [required] [default: "**/*"]

Options:
  --version       Show version number                                  [boolean]
  --config        Path to JSON config file
  --forward-args  Args that should be forwarded to the codemod
                                                           [array] [default: []]
  --cwd           The directory containing the project's node_modules directory
               [string] [default: "/home/varl/dev/dhis2/cli/cli-utils-codemods"]
  -h, --help      Show help                                            [boolean]

Missing required positional arguments (got 1 of 2)
```

If I add a positional the commands runs:

```
varl@veronika:~/dev/dhis2/cli/cli-utils-codemods$ ./bin/d2-utils-codemods apply /home/varl/dev/dhis2/libs/d2/codemods/31.1.0-change-imports.js src/
Processing 12 files...
Spawning 3 workers...
Sending 4 files to free worker...
Sending 4 files to free worker...
Sending 4 files to free worker...
All done.
Results:
0 errors
8 unmodified
4 skipped
0 ok
Time elapsed: 0.177seconds
```

The `<files...>` in the "command" string on L3 means it is required, so if we change that to `[files...]` it works as expected:

```suggestion
module.exports.command = 'apply <codemod> [files..]'
```

Reply:
Changed to the optional argument syntax